### PR TITLE
[Docs] Improve MM coverage

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

**Review by gilad-shaham — APPROVED** [review]

File: docs/tutorials/genai-02-model-monitor-llm.ipynb [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96df62bc-555c-4700-b118-65b0ccacb913",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile monit-code.py\n",
+    "import enum\n",
```
Comment (gilad-shaham) [inline docs/tutorials/genai-02-model-monitor-llm.ipynb]: Remove import

File: docs/tutorials/genai-02-model-monitor-llm.ipynb [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+    "import re\n",
+    "from typing import Any, Union\n",
+    "\n",
+    "import mlrun\n",
+    "import mlrun.common.schemas\n",
+    "from mlrun.model import ModelObj\n",
```
Comment (gilad-shaham) [inline docs/tutorials/genai-02-model-monitor-llm.ipynb]: Remove import

File: docs/concepts/model-monitoring.md [hunk 3]
```diff
@@ -32,9 +32,14 @@ Model monitoring is currently in Tech Preview status. Migrating from 1.7.0 to la
...
 ../model-monitoring/applications
+../model-monitoring/monitoring-models
+../model-monitoring/monitoring-models-grafana
+../model-monitoring/index
+../concepts/alerts
 ```
+
+**See also**
+
+- {ref}`genai-02-mm-llm`
```
Comment (assaf758) [inline docs/concepts/model-monitoring.md:44]: i think this demo is removed, no?

File: docs/genai/deployment/index.md [hunk 4]
```diff
@@ -17,5 +17,7 @@ genai_serving_graph
 
 **See also**
 - {ref}`genai_01_basic_tutorial`
-- {ref}`model-monitoring`
+- {ref}`genai-02-mm-llm`
```
Comment (assaf758) [inline docs/genai/deployment/index.md:20]: same - please remove everywhere

**Review by assaf758 — CHANGES_REQUESTED** [review]

File: docs/genai/deployment/index.md (hunk 4 above)
Comment (jillnogold) [reply docs/genai/deployment/index.md:20]: This is a valid link to a new tutorial: genai-02-model-monitor-llm.ipynb

File: docs/concepts/model-monitoring.md (hunk 3 above)
Comment (jillnogold) [reply docs/concepts/model-monitoring.md:44]: This is a valid link to a new tutorial: genai-02-model-monitor-llm.ipynb

**Review by assaf758 — APPROVED** [review]

**Review by assaf758 — APPROVED** [review]

**Pull request merged**